checkrebuild

```diff
@@ -22,17 +22,26 @@ get_unofficial_pkgs() {
     comm -23 <(pacman -Qq | sort) <(pacman -Sql "${repos_to_skip[@]}" 2>/dev/null | sort)
 }
 
-filter_executable() {
-    while read -r file; do
-        [[ -f "$file" && -x "$file" ]] && echo "$file"
-    done
+newline2null(){
+    perl -pe 's/\n/\0/'
 }
 
 get_package_files() {
     xargs pacman -Qql |
-    perl -pe 's/\n/\0/' |
-    xargs -0 readlink -f |
-    filter_executable
+    newline2null |
+    xargs -0 readlink -e |
+    sort -u
```

Comment:
I think this can preserve nul-byte all the way, right?

```
xargs -0 readlink -ez |
sort -uz
```

Reply:
Neat, if only `xargs` got `-z` switch.